show/main.py

```diff
@@ -172,8 +171,8 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
-# 'summary' subcommand ("show interfaces summary")
-@interfaces.command()
+# 'summary' subcommand ("show interfaces summary") -- called if no subcommands are passed
+@interfaces.command(default=True)
```

Comment:
I am not a fan of the current DefaultGroup we have built around Click. If a group has a default command, using Tab to show available subcommands does not work.

We either need to remove the DefaultGroup concept altogether, or fix the DefaultGroup to allow the Tab functionality to work just as it does for other commands.